### DISCUSSION TITLE
Bots with decon order break under certain conditions

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13919] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When deconstructing items, bots with the Deconstruct Items order seem to break - pretty much exactly like what `disablecrewai` does. It seems to happen while they are deconstructing items (they just stand at the deconstructor, and ignore any other orders they may have) or if another bot is deconstructing something (even if they are on the other side of the submarine they will freeze in place while the deconstructor is running). This will persist until the items are finished deconstructing, which may not happen if the deconstructor is unpowered or breaks, leading to the bots remaining stuck indefinitely until you remove the order or fix the deconstructor.

https://github.com/FakeFishGames/Barotrauma/assets/37850646/3729a2b0-a3e7-4be4-9838-2c0bf2a9a9bb

### Reproduction steps

1. spawn several items for bots to deconstruct
2. spawn some bots and give them all the order to deconstruct as their highest priority order, then any other orders after that.
3. observe that they will put some items into the deconstructor, then all of them will pause and wait for the items to finish before doing anything else

### Version

v1.4.6.0 hotfix 2 unstable

**Comment by itchyOwl** [comment]
Can't repro. Where those items were spawned? Are they in the deconstructor when they get stuck? Would help if you could enable "debugai" for the video.

**Reply by Vykn** [reply]
Works with any deconstructable item. Just get a few stacks, mark them to be decon'd and drop them on the ground. The bots will load up the deconstructor, and sit and wait for it to finish before doing anything else.

https://github.com/FakeFishGames/Barotrauma/assets/37850646/bded7504-0ede-4c88-b2ba-990e88433d58

Spawned in 24 fuel rods, marked them for deconstruction, dropped them on the ground. The bots sit there, neglecting their other orders.
